packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -422,7 +429,14 @@ const handleOriginResponse = async ({
       Bucket: bucketName,
       Key: `static-pages${hasFallback.fallback}`
     };
-    const { Body } = await s3.getObject(s3Params).promise();
+    const { GetObjectCommand } = await import(
+      "@aws-sdk/client-s3/commands/GetObjectCommand"
+    );
+    const { Body } = await s3.send(new GetObjectCommand(s3Params));
+
+    // Body is stream per: https://github.com/aws/aws-sdk-js-v3/issues/1096
+    const getStream = await import("get-stream");
```

Comment:
I am using get-stream package: https://bundlephobia.com/result?p=get-stream@6.0.0 which is small